INSTALL.md

```diff
@@ -21,6 +21,7 @@ mkdir samples
 vim .env
 docker compose up --scale daemon=3  # this will take a while
 docker compose exec web python3 -m mquery.db
+docker compose exec web python3 -m alembic upgrade head
```

Comment:
Not sure about this. We want to reduce the number of additional commands to zero (#402), not add more of them.

Reply:
If you don't want to fix this in this PR and want to work separately in #402, it's OK to leave it as it is now (i.e. don't change the documentation in this PR. It works for fresh installations, just doesn't run migrations automatically).

Reply:
> Not sure about this. We want to reduce the number of additional commands to zero (#402), not add more of them.

All because of alembic, which does not automatically recognize the modification from varchar to enum. I am able to create an enum type in postgres from the model level, but alembic does not replace it (it still remains VARCHAR) - that's why my manual modifications in migration files.

I found a library https://pypi.org/project/alembic-postgresql-enum/, quite new, I have to test it, but the question is whether to enter something like that at all. Or maybe look for solutions in sqlalchemy dialects for postgresql

Reply:
don't change the documentation. After #421 alembic will be run with web automatically anyway.